lib/her/model/orm.rb

```
@@ -75,6 +75,7 @@ def destroy
         method = self.class.method_for(:destroy)
         run_callbacks :destroy do
           self.class.request(:_method => method, :_path => request_path) do |parsed_data, response|
+            raise Her::Errors::RemoteResourceError if response.status >= 400
```

so this is including validation-y error codes because we think those should just be considered failures? should we surface the errors in that case?